addons/core/translations/resources/en-us.yaml

```diff
@@ -954,6 +954,7 @@ policy:
   messages:
     detach: Successfully detached
     reapply: Successfully reapplied
+    later: The policy will be reapplied after the recording has completed
```

Comment:
Where do we use this translation?

Reply:
this is for a new change I will making in this PR

Reply:
I'm going to move this change to a separate PR since that is not a bug that can wait a bit